src/components/Icon/icons/ArticleList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconColorsMap from './colors';
+
+const ArticleList = ({ fill, ...props }) => (
+  <svg width="24" height="24" viewBox="0 0 24 24" fill={fill} {...props}>
```

Comment:
For all of our SVGs, we have run them through `svgo` to optimize every SVG in the library. One tool you can quickly use for this ishttps://jakearchibald.github.io/svgomg. You also need to include the `xmlns="http://www.w3.org/2000/svg"` attribute for consistency.